### ISSUE TITLE
Monitor for faults at standby

### ISSUE THREAD

**Issue by spinler** [openbmc/openbmc#1731]
At standby, we need to watch for VIN_UV_FAULT from STATUS_WORD, and create the appropriate error logs and also update that status somewhere, like an inventory property on a power supply interface on the power supply.

Either polling every second, or interrupt driven somehow.

**Comment by bjwyman** [comment]
[ ] Create a power supply monitoring application (phosphor-power-psu?)
[ ] Monitor STATUS_WORD and VIN_UV_FAULT values, create error log if bad values.
[ ] Update recipe and service files to start monitor application at standby

**Comment by bjwyman** [comment]
STATUS
07/13: 
* On hold

**Comment by bjwyman** [comment]
Final/result and testing will depend on openbmc/openbmc#1883.

**Comment by williamspatrick** [comment]
Cannot commit to sprint 17.07.26 because of missing dependency on device driver.

**Comment by rfrandse** [comment]
<https://gerrit.openbmc-project.xyz/6018> Power supply monitor recipe, service, and conf
Resolves:  #1731 Monitor for faults at standby

**Comment by bjwyman** [comment]
This probably will not work good. @bradbishop @geissonator 
https://gerrit.openbmc-project.xyz/#/c/6018/
That depends on the updates in the witherspoon-pfault-analysis repo, which did not get merged.

Depends-On: Ie364a3dc5b7f19cbc587450fc54f231d1ec280f7

These need +2 and merge:
https://gerrit.openbmc-project.xyz/5527
https://gerrit.openbmc-project.xyz/5714
https://gerrit.openbmc-project.xyz/5861
https://gerrit.openbmc-project.xyz/5917
https://gerrit.openbmc-project.xyz/5961
https://gerrit.openbmc-project.xyz/6047
https://gerrit.openbmc-project.xyz/6048

**Comment by spinler** [comment]
Not actually complete.

**Comment by spinler** [comment]
Now complete.
